packages/flutter/lib/fix_data.yaml

```diff
@@ -11,6 +11,28 @@
 version: 1
 transforms:
 
+  # Changes made in https://github.com/flutter/flutter/pull/45941
+  - title: 'Rename to deferFirstFrame'
+    date: 2020-12-23
+    element:
+      uris: [ 'material.dart' ]
```

Comment:
You also want this fix is you import widgets.dart directly, (or if you import it via cupertino.dart)